src/jit/hwintrinsicArm64.h

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef _HW_INTIRNSIC_ARM64_H_
+#define _HW_INTIRNSIC_ARM64_H_
+
+#if FEATURE_HW_INTRINSICS
+
+struct HWIntrinsicInfo
+{
+    enum Form
+    {
+        IsSupported,
```

Comment:
Each of these should have a short comment describing them. Also, I'm not sure what the `Unsupported` value of `Form` would be used for.